README.md

```diff
@@ -1,149 +1,43 @@
-# Getting started example for Mbed OS
+![](./resources/official_armmbed_example_badge.png)
+# Blinky Mbed OS Example
 
-This guide reviews the steps required to get Blinky with the addition of dynamic OS statistics working on an Mbed OS platform. (Note: To see a rendered example you can import into the Arm Online Compiler, please see our [quick start](https://os.mbed.com/docs/mbed-os/latest/quick-start/online-with-the-online-compiler.html#importing-the-code).)
+The example project is part of the [Arm Mbed OS quick start guide](https://os.mbed.com/docs/mbed-os/v5.14/quick-start/index.html). It contains an application to repeatedly blink an LED on supported [Mbed boards](https://os.mbed.com/platforms/).
```

Comment:
I would say that the example is part of the official Mbed OS examples and then say that it is the getting started example for Mbed OS.

```
The example project is part of the [Mbed OS Official Examples](https://os.mbed.com/code/) and is the [getting started example for Mbed OS](https://os.mbed.com/docs/mbed-os/v5.14/quick-start/index.html). It contains an application that repeatedly blinks an LED on supported [Mbed boards](https://os.mbed.com/platforms/).
```

Reply:
[Done](https://github.com/ARMmbed/mbed-os-example-blinky/pull/192/commits/effcefb11afb9af5c5117b0db341a1d7b982ebf5).